src/commands/test-branch.yml

```diff
@@ -57,6 +57,17 @@ steps:
       name: "Solidus <<parameters.branch>>: Store Capybara screenshots"
       path: spec/dummy/tmp/capybara
       destination: screenshots/capybara
+  - run:
+      name: 'Drop database'
+      environment:
+        SOLIDUS_BRANCH: <<parameters.branch>>
+      command: |
+        if [[ -d "spec/dummy" ]]; then
```

Comment:
I added this condition to allow this orb's CI build to skip dropping the databaase. However, I've been able to confirm that the command is working in https://app.circleci.com/pipelines/github/solidusio-contrib/solidus_product_assembly/302/workflows/8b5b750d-f932-42af-8654-0aa00e373f2d.

If we want to test dropping the database, I think the simplest way to do that is to generate a simple Solidus extension and commit it to this repo. Let me know if you'd think we should go with that approach.

Reply:
Improving the tests here would be awesome 🙌 – but not blocking for this PR, as we need to fix the product-assembly failures, can you work on that in another PR?

Reply:
> can you work on that in another PR?

@elia You mean improving the tests? Yes, I can work on that on a separate PR :)

As for the `product_assembly` extension, I think we'll need to merge this PR in order for `product_assembly` (and other extensions) to use this updated CircleCI orb.